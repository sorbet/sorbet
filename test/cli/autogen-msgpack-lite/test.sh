#!/bin/bash
set -eu

main/sorbet --silence-dev-message --stop-after namer --autogen-version=6 \
  -p autogen:autogen.txt -p autogen-msgpack:autogen.msgpack \
  --autogen-msgpack-skip-reference-metadata \
  --remove-path-prefix=test/cli/ \
  test/cli/autogen-msgpack-lite \
  test/cli/autogen-msgpack-lite/file_with_no_dot

echo "--- autogen.txt start ---"
cat autogen.txt
echo "--- autogen.txt end ---"
echo

shasum autogen.msgpack

