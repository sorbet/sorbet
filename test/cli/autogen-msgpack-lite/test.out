--- autogen.txt start ---
# ParsedFile: test/cli/autogen-path-prefix/file_with_no_dot
requires: []
## defs:
[def id=0]
 type=module
 defines_behavior=0
 is_empty=0
## refs:
[ref id=0]
 scope=[]
 name=[HotdogGenerator]
 nesting=[]
 resolved=[HotdogGenerator]
 loc=autogen-path-prefix/file_with_no_dot:8
 is_defining_ref=0
[ref id=1]
 scope=[]
 name=[HotdogConsumer]
 nesting=[]
 resolved=[HotdogConsumer]
 loc=autogen-path-prefix/file_with_no_dot:10
 is_defining_ref=0
# ParsedFile: test/cli/autogen-path-prefix/hotdog_consumer.rb
requires: []
## defs:
[def id=0]
 type=module
 defines_behavior=0
 is_empty=0
[def id=1]
 type=module
 defines_behavior=1
 is_empty=0
 defining_ref=[HotdogConsumer]
## refs:
[ref id=0]
 scope=[]
 name=[HotdogConsumer]
 nesting=[]
 resolved=[HotdogConsumer]
 loc=autogen-path-prefix/hotdog_consumer.rb:3
 is_defining_ref=1
# ParsedFile: test/cli/autogen-path-prefix/hotdog_generator.rb
requires: []
## defs:
[def id=0]
 type=module
 defines_behavior=0
 is_empty=0
[def id=1]
 type=module
 defines_behavior=1
 is_empty=0
 defining_ref=[HotdogGenerator]
## refs:
[ref id=0]
 scope=[]
 name=[HotdogGenerator]
 nesting=[]
 resolved=[HotdogGenerator]
 loc=autogen-path-prefix/hotdog_generator.rb:3
 is_defining_ref=1
--- autogen.txt end ---

6dbf3772d0c8000f9c773ab3bd0e5923c27d241a  autogen.msgpack
